app/models/dog.rb

```diff
@@ -0,0 +1,11 @@
+class Dog < ActiveRecord::Base
+  belongs_to :user
+
+  has_attached_file :img,
+    styles: {medium: '300x300>'},
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.